x-pack/plugins/apm/public/components/app/correlations/ml_latency_correlations.tsx

```diff
@@ -0,0 +1,428 @@
+  }, []);
+
+  useEffect(() => {
+    if (isErrorMessage(error)) {
+      notifications.toasts.addDanger({
+        title: i18n.translate(
+          'xpack.apm.correlations.latencyCorrelations.errorTitle',
+          {
+            defaultMessage: 'An error occurred fetching correlations',
+          }
+        ),
+        text: error.toString(),
+      });
+    }
+  }, [error, notifications.toasts]);
+
+  const [
+    selectedSignificantTerm,
+    setSelectedSignificantTerm,
+  ] = useState<SelectedSignificantTerm | null>(null);
+
+  let selectedHistogram = histograms.length > 0 ? histograms[0] : undefined;
+
+  if (histograms.length > 0 && selectedSignificantTerm !== null) {
+    selectedHistogram = histograms.find(
+      (h) =>
+        h.field === selectedSignificantTerm.fieldName &&
+        h.value === selectedSignificantTerm.fieldValue
+    );
+  }
+  const history = useHistory();
+  const trackApmEvent = useUiTracker({ app: 'apm' });
+
+  const mlCorrelationColumns: Array<
+    EuiBasicTableColumn<MlCorrelationsTerms>
+  > = useMemo(
+    () => [
+      {
+        width: '116px',
+        field: 'correlation',
+        name: (
+          <EuiToolTip
+            content={i18n.translate(
+              'xpack.apm.correlations.latencyCorrelations.correlationsTable.correlationColumnDescription',
+              {
+                defaultMessage:
+                  'The impact of a field on the latency of the service, ranging from 0 to 1.',
+              }
+            )}
+          >
+            <>
+              {i18n.translate(
+                'xpack.apm.correlations.latencyCorrelations.correlationsTable.correlationLabel',
+                {
+                  defaultMessage: 'Correlation',
+                }
+              )}
+              <EuiIcon size="s" color="subdued" type="questionInCircle" />
+            </>
+          </EuiToolTip>
+        ),
+        render: (_: any, term: MlCorrelationsTerms) => {
+          return <div>{asPreciseDecimal(term.correlation, 2)}</div>;
+        },
+      },
+      {
+        field: 'fieldName',
+        name: i18n.translate(
+          'xpack.apm.correlations.latencyCorrelations.correlationsTable.fieldNameLabel',
+          { defaultMessage: 'Field name' }
+        ),
+      },
+      {
+        field: 'fieldValue',
+        name: i18n.translate(
+          'xpack.apm.correlations.latencyCorrelations.correlationsTable.fieldValueLabel',
+          { defaultMessage: 'Field value' }
+        ),
+        render: (_: any, term: MlCorrelationsTerms) =>
+          String(term.fieldValue).slice(0, 50),
+      },
+      {
+        field: 'duplicatedFields',
+        name: i18n.translate(
+          'xpack.apm.correlations.latencyCorrelations.correlationsTable.duplicatesLabel',
+          { defaultMessage: 'Duplicates' }
+        ),
+        render: (_: any, term: MlCorrelationsTerms) => {
+          return (
+            <div style={{ display: 'flex', flexDirection: 'column' }}>
```

Comment:
did you try `EuiFlexGroup`?

Reply:
Updated here https://github.com/elastic/kibana/pull/99905/commits/fdeafd869ab95e17cc81598d1633a646415e9f38